fuzzing/private/binary.bzl

```diff
@@ -36,6 +36,7 @@ Provider for storing information about a fuzz test binary.
         "corpus_dir": "The directory of the corpus files used as input seeds.",
         "dictionary_file": "The dictionary file to use in fuzzing runs.",
         "engine_info": "The `FuzzingEngineInfo` provider of the fuzzing engine used in the fuzz test.",
+        "workspace_name": "The name of the workspace of the fuzz test executable.",
```

Comment:
Assuming this field is actually needed (see my other comment), I think this name is non-specific and inconsistent with the other fields. How about calling it `binary_workspace` and placing it along the other `binary_*` fields in this structure?